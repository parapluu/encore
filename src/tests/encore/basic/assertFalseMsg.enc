-- This file was automatically converted by encorec

class Main
  def main() : void
    assertFalse(false)
    assertFalse(true, "Int {} Bool {} String {}", 42, false, "Foo")
  end
end

