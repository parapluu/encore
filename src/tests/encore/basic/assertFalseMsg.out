Assertion failed at "assertFalseMsg.enc" (line 4, column 5):
Int 42 Bool false String Foo
